spec/factories/hearing_types.rb

```diff
@@ -2,7 +2,16 @@
 
 FactoryBot.define do
   factory :hearing_type do
-    description { 'This is a description' }
+    description do
+      [
```

Comment:
While this may work nicely in the UI, I think the examples here are two parts description and code (PTP)